desktop/assets/texts/es-ES/settings.ftl

```diff
@@ -3,6 +3,8 @@ player-settings = Configuración de reproducción
 movie-parameters = Parámetros de película
 custom-base-url = URL Base personalizada
 spoof-swf-url = Falsificar URL de SWF
+referer-url = URL de origen
+cookie = Cookie
```

Comment:
FYI (for all languages), these are technical terms related to HTTP here:
https://en.wikipedia.org/wiki/HTTP_referer
https://en.wikipedia.org/wiki/HTTP_cookie

Reply:
If we're going to uphold the idea that technical terms should not be translated, then I would advocate not injecting those strings at all. It's a little disingenuous and to be honest, I've always felt dubious about keeping technical terms in English for Ruffle because not all of our audience are the most technically inclined.

Reply:
> If we're going to uphold the idea that technical terms should not be translated

I'm not saying that. I referred (hah, pun intended) to ` URL de origen` mostly - I have don't speak Spanish, so I have no idea how it's used in Spanish, so I can accept this. I was just pointing this out "just in case" there is a different word for it in this context.

Reply:
Spanish is not a very robust language when it comes to modern technical terms. It mostly just borrows from English. Even the terms that you can translate tend to not be used that way because a lot of them just articulate really awkwardly :P